### ISSUE TITLE
Add support for purl?

### ISSUE THREAD

**Issue by pombredanne** [anthonyharrison/csaf#1]
Hi: I was wondering if you have plans to add support for package-url / purl as supported in the CSAF spec? 
See https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31334-full-product-name-type---product-identification-helper---purl
Thanks!

**Comment by anthonyharrison** [comment]
@pombredanne Yes I will be adding PURL support. However in all of the CSAF documents I have seen so far, none have used purl preferring to use 'cpe' instead.
